ci/test/05_before_script.sh

```diff
@@ -48,6 +48,7 @@ if [ -z "$NO_DEPENDS" ]; then
 fi
 if [ -n "$PREVIOUS_RELEASES_TO_DOWNLOAD" ]; then
   BEGIN_FOLD previous-versions
+  DOCKER_EXEC gpg --list-keys $MAINTAINER_KEY >/dev/null 2>&1 || gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $MAINTAINER_KEY
```

Comment:
```suggestion
  DOCKER_EXEC gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $MAINTAINER_KEY
```

Would be surprising if a vanilla docker image had the key already

Reply:
Maybe we want to cache it though if there's a DDOS on PGP key servers again.